src/hotspot/share/runtime/os.hpp

```diff
@@ -229,6 +229,7 @@ class os: AllStatic {
   // Get summary strings for system information in buffer provided
   static void  get_summary_cpu_info(char* buf, size_t buflen);
   static void  get_summary_os_info(char* buf, size_t buflen);
+  static ssize_t pd_write(int fd, const void *buf, size_t nBytes);
```

Comment:
What is the required meaning of the return value here? I'm assuming < 0 -> error; while >= 0 -> bytes written?

Reply:
Return value changed to `bool`. All calls changed accordingly.